Description: Temporarily disable hyper-rustls to avoid even more rustls patches
Forwarded: not-needed
Last-Update: 2024-11-01
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -215,8 +215,8 @@
-__rustls = [
-    "dep:hyper-rustls",
-    "dep:tokio-rustls",
-    "dep:rustls",
-    "__tls",
-    "dep:rustls-pemfile",
-    "dep:rustls-pki-types",
-]
+#__rustls = [
+#    "dep:hyper-rustls",
+#    "dep:tokio-rustls",
+#    "dep:rustls",
+#    "__tls",
+#    "dep:rustls-pemfile",
+#    "dep:rustls-pki-types",
+#]
@@ -224,2 +224,2 @@
-    "hyper-rustls?/ring",
-    "tokio-rustls?/ring",
+#    "hyper-rustls?/ring",
+#    "tokio-rustls?/ring",
@@ -277 +277 @@
-    "hyper-rustls?/http2",
+#    "hyper-rustls?/http2",
@@ -305 +305 @@
-rustls-tls-manual-roots-no-provider = ["__rustls"]
+rustls-tls-manual-roots-no-provider = []#["__rustls"]
@@ -312,2 +312,2 @@
-    "hyper-rustls?/native-tokio",
-    "__rustls",
+#    "hyper-rustls?/native-tokio",
+#    "__rustls",
@@ -391,8 +391,8 @@
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.hyper-rustls]
-version = "0.27.0"
-features = [
-    "http1",
-    "tls12",
-]
-optional = true
-default-features = false
+#[target.'cfg(not(target_arch = "wasm32"))'.dependencies.hyper-rustls]
+#version = "0.27.0"
+#features = [
+#    "http1",
+#    "tls12",
+#]
+#optional = true
+#default-features = false
@@ -462,4 +462,4 @@
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.rustls-pki-types]
-version = "1.1.0"
-features = ["alloc"]
-optional = true
+#[target.'cfg(not(target_arch = "wasm32"))'.dependencies.rustls-pki-types]
+#version = "1.1.0"
+#features = ["alloc"]
+#optional = true
@@ -483,5 +483,5 @@
-[target.'cfg(not(target_arch = "wasm32"))'.dependencies.tokio-rustls]
-version = "0.26"
-features = ["tls12"]
-optional = true
-default-features = false
+#[target.'cfg(not(target_arch = "wasm32"))'.dependencies.tokio-rustls]
+#version = "0.26"
+#features = ["tls12"]
+#optional = true
+#default-features = false
